vcs/vcsvtk/pipeline1d.py

```diff
@@ -103,6 +103,7 @@ def plot(self, data1, data2, tmpl, grid, transform, **kargs):
                 self._context().canvas.plot(m, donotstoredisplay=True)
 
         ren2 = self._context().createRenderer()
+        ren2.SetLayer(1)
```

Comment:
@doutriaux1 You are right. 
Something like this:
self._context().setLayer(ren2, tmpl.data.priority)
What is object.priority?